<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Company</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 43, 36);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_insert_emploees">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Sitka Small&quot;;
color: rgb(255, 255, 255);
background-color: rgb(43, 43, 43);</string>
       </property>
       <property name="text">
        <string>افزودن یک کارمند جدید</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_edit_emploees">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Sitka Small&quot;;
color: rgb(255, 255, 255);
background-color: rgb(43, 43, 43);</string>
       </property>
       <property name="text">
        <string>ویرایش مشخصات کارمند</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_list_emploees">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Sitka Small&quot;;
color: rgb(255, 255, 255);
background-color: rgb(43, 43, 43);</string>
       </property>
       <property name="text">
        <string>لیست کارمندان</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
